services/iplant-groups/src/iplant_groups/clients/grouper.clj

```diff
@@ -98,21 +98,21 @@
 ;; Group retrieval.
 
 (defn- group-retrieval-query-filter
-  [group-id]
-  (remove-vals nil? {:groupUuid       group-id
-                     :queryFilterType "FIND_BY_GROUP_UUID"}))
+  [group-name]
+  (remove-vals nil? {:groupName       group-name
+                     :queryFilterType "FIND_BY_GROUP_NAME_EXACT"}))
```

Comment:
Many of the changes to this file allowed us to switch from using UUIDs in URI paths to using names instead, which is much more convenient for client services.